CONTRIBUTING.md

```diff
@@ -57,3 +88,7 @@ To regenerate the `/docs` Markdown run:
 ```sh
 go generate
 ```
+
+## Sign Your Commits
+
+To get your PR merged, we require you to sign your commits. Fortunately, this has become very easy to [set up](https://developer.1password.com/docs/ssh/git-commit-signing/)!
```

Comment:
I think here we should be a bit less opinionated and add other alternatives like so:

<details>
<summary><b>Sign commits with <code>gpg</code></b></summary>
<p></p>

Follow the steps below to set up commit signing with `gpg`:
1. [Generate a GPG key](https://docs.github.com/en/authentication/managing-commit-signature-verification/generating-a-new-gpg-key)
2. [Add the GPG key to your GitHub account](https://docs.github.com/en/authentication/managing-commit-signature-verification/adding-a-gpg-key-to-your-github-account)
3. [Configure `git` to use your GPG key for commit signing](https://docs.github.com/en/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key#telling-git-about-your-gpg-key)

</details>

<details>
<summary><b>Sign commits with <code>ssh-agent</code></b></summary>
<p></p>

Follow the steps below to set up commit signing with `ssh-agent`:
1. [Generate an SSH key and add it to `ssh-agent`](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)
2. [Add the SSH key to your GitHub account](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account)
3. [Configure `git` to use your SSH key for commit signing](https://docs.github.com/en/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key#telling-git-about-your-ssh-key)

</details>

<details>
<summary><b>Sign commits with 1Password</b></summary>
<p></p>

You can also sign commits using 1Password, which lets you sign commits with biometrics without the signing key leaving the local 1Password process.

Learn how to [use 1Password to sign your commits](https://developer.1password.com/docs/ssh/git-commit-signing/).

<a href="https://www.youtube.com/watch?feature=player_embedded&v=BMFvhl0WRFQ" target="_blank">
  <img src="https://img.youtube.com/vi/BMFvhl0WRFQ/hqdefault.jpg" alt="Watch the demo" width=380 />
</a>

</details>

This was extracted from [`check-signed-commits-action`](https://github.com/1Password/check-signed-commits-action) -> `action.yml`

We can reorder them if we want to make the 1Password option come first 😄

Reply:
Great suggestion! Updated it.